modules/microshift-config-parameters-table.adoc

```diff
@@ -139,14 +161,50 @@ Not all MIME types benefit from compression, but `HAProxy` uses resources to try
 |`443`
 |Default port shown. Configurable. Valid value is a single, unique port in the `1-65535` range. The values of the `ports.http` and `ports.https` fields cannot be the same.
 
-|`ingress.routeAdmissionPolicy.namespaceOwnership`
+//Has this parameter been renamed to 'ingress.routeAdmission'?
```

Comment:
No, see https://github.com/openshift/microshift/blob/main/packaging/microshift/config.yaml#L170; this parameter is still present and should not be deleted unless there is an engineering PR inflight that changes it.